portal/server/routes/utilisation-report-service/utilisation-report-download/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const { validateBankIdForUser, validateMongoId, validateToken, validateRole } = require('../../middleware');
+const { PAYMENT_REPORT_OFFICER } = require('../../../constants/roles');
+const { getReportDownload } = require('../../../controllers/utilisation-report-service');
+
+const router = express.Router();
+
+router.get(
+  'banks/:bankId/utilisation-report-download/:_id',
+  [validateToken, validateRole({ role: [PAYMENT_REPORT_OFFICER] })],
+  validateBankIdForUser,
+  validateMongoId,
```

Comment:
None of these validators currently use `express-validator` so `handleValidation` isn't currently need. We don't currently have thehave the `handleValidation` function in this project, and I guess this is because we're in the UI layer so wouldn't just return a `400` status response along with the errors. We would instead need to do a redirect, or render an error page, and this might be different depending on the particular validation handler. What do you think?

Reply:
Ah I've just seen validateMongoId and us in portal not portal-api. As it's just all going to not found and we don't need good errors that's fine - ignore my previous comment.
Could I get you to add in a console.warn or something on both the mongoId/bankId validation so we have a record of it happening in logs please?